x/emissions/types/codec.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/types/msgservice"
 )
 
-func RegisterCodec(cdc *codec.LegacyAmino) {
+func RegisterCodec(_ *codec.LegacyAmino) {
```

Comment:
I didn't check it yet, but I have a bad feeling about ignoring Amino registration.

Ledger devices still use amino in the Cosmos SDK, so if anyone uses Ledger and amino is not implemented, then Ledger devices will not be able to sign.

If anyone checked already, please leave a note here. Otherwise, I'll check if I have some time.

Reply:
Thanks for pointing this out.
I think we just have this method empty because `emissions` modules don't implement messages. We have the body defined for the other modules.